components/BrownBanner.tsx

```diff
@@ -12,7 +13,8 @@ export const BrownBanner: React.FC = () => {
           <Image src={OITLogo} alt="OIT Logo" width={200} height={0} priority />
         </a>
         <div className="ml-6 w-full flex flex-row justify-between items-center">
-          <h1 className="py-7 pl-6 h-full sm:text-xl lg:text-2xl font-semibold border-l border-black md:block hidden md:visible">
+          <h1 className="py-7 pl-6 h-full sm:text-xl lg:text-2xl font-semibold border-l border-black hidden md:flex items-center md:visible space-x-4">
+            <CBCLogo width={50} />
             Computational Biology Core
           </h1>
```

Comment:
RE: Your question about `space-x-4`

`space-x-4` works on the items of a container (div is typical). The "Computational Biology Core" text wasn't in it's own html tags. Here is the code suggestion that I think gets at what you need ! 

```suggestion
          <div className="py-7 pl-6 h-full sm:text-xl lg:text-2xl font-semibold border-l border-black hidden md:flex items-center md:visible space-x-4">
            <CBCLogo width={50} />
            <h1>Computational Biology Core</h1>
          </div>
```

Reply:
Thank you!!! This was driving me crazy yesterday. 🤪